### ISSUE TITLE
regression_loss

### ISSUE THREAD

**Issue by jzpcn** [zylo117/Yet-Another-EfficientDet-Pytorch#728]
regression_loss = torch.where(
                    torch.le(regression_diff, 1.0 / 9.0),
                    0.5 * 9.0 * torch.pow(regression_diff, 2),
                    regression_diff - 0.5 / 9.0
                )

why you use this method to compute reg_loss rather than the normal smooth L1 loss
